### DISCUSSION TITLE
Documentation for pyjab module

### DISCUSSION THREAD

**Discussion by niteshkumar2000** [gaozhao1989/pyjab#67] [Q&A]
Hello, just wanted to know if there's documentation available for available set of APIs or where I could find them?
It'll be good even if I get some example code to see how the module works! Thanks :)

**Comment by gaozhao1989** [comment]
Hi,
There is no document for pyjab now. Please check with wiki for few samples and thanks.

**Reply by niteshkumar2000** [reply]
Can you please give me a link to that? I can't seem to find it. Thanks :)

**Reply by gaozhao1989** [reply]
Check with this

https://github.com/gaozhao1989/pyjab/wiki

**Reply by niteshkumar2000** [reply]
Thanks for sharing the link, I'm wondering how to find a dialog element and elements inside that, when I click a button it pops up a dialog, the given example code is unable to find that. Can you help me with this? Thanks a lot.

**Reply by gaozhao1989** [reply]
Hi,

Highly advice you need to check the new window is different java application instance or not. If that you need to create new driver instance.

**Reply by niteshkumar2000** [reply]
It's the same application but a modal dialog. I tried finding it by name, but I'm not able to find it. What other ways are there to achieve this?

**Reply by gaozhao1989** [reply]
Check with [Access Bridge Explorer](https://github.com/google/access-bridge-explorer) if the new window with in same Java instance

**Reply by niteshkumar2000** [reply]
Hi, I checked with Access Bridge Explorer, it's under the same JVM instance but the code is unable to find the dialog instance. If I remove the outer frame code it's working fine. What could be the reason?

**Reply by gabrielnuss** [reply]
i manage to resolve it, my problem was acessing a action button and calling the function "click", when i did this it didn't found the window that opens when i click it, but if instead of calling click function i do this: "actiobButton.sned_text(" ", true)", it works fine even if other window is in front of it.

**Reply by xjpaulo** [reply]
> Hi, I checked with Access Bridge Explorer, it's under the same JVM instance but the code is unable to find the dialog instance. If I remove the outer frame code it's working fine. What could be the reason?

Hi, how did you solve the problem?
